en/docs/wip/api-recommendations.md

```diff
@@ -0,0 +1,92 @@
+
+# AI based Recommendation System for the API Developer portal
+
+## API Marketplace
+
+You must be  familiar with different marketplaces such as Ebay, Amazon or AliExpress. 
+There are two types of users in these marketplaces. Sellers and Consumers. 
+Sellers market their products in the marketplace by publishing details of the product and the price. 
+Consumers consume these products after selecting the appropriate product based on their needs. 
+This concept is equally valid for API domain. 
+
+WSO2 API Manager Developer Portal is a marketplace for APIs. API publishers (Sellers) will publish their APIs 
+through the Publisher and those APIs will be listed in the developer portal. Application Developers (Consumers) can 
+login to the portal to choose and subscribe which APIs to leverage in their applications. The only difference between 
+a product marketplace and an API marketplace is that, the products in API marketplaces (APIs) are constantly updating 
+with new features, new capabilities and new use cases. How successful are we, when marketing a new feature of an API 
+to the correct audience, when there are thousands of APIs in the marketplace?
+
+Solution-seeking App developers know that they want APIs for their application, but they aren’t sure what the best 
+solution is for their needs. There are thousands of different APIs and the developer has to pick the APIs of interest 
+by browsing through the APIs or searching with a particular information associated with the API. 
+
+With this feature, we are addressing the two issues mentioned above.  
+
+- Find the best audience for the API, whom will be really interested in the APIs capabilities and use it in their
+ applications,
+ - Recommending APIs based on the application developers interests and the application that they are creating.
+
+Using AI technologies, this feature will analyze the behavior of the developer, application that he or she is 
+developing and suggest a list of new APIs that are not subscribed by the user but might be useful for the project. 
+In order to recommend effectively, there must be at least 20 APIs in the portal.
+
+![](../assets/img/api-recommendations.png)
+
+## Solution
+Recommendation server will be deployed in WSO2 and will be provided to all the customers as a service. This service 
```

Comment:
Recommendation service is available as a SaaS service for all wso2 customers and secured via oauth2. If a specific customer requires to enable this feature, they will have to request a consumer key and secret via the wso2 support portal.

Reply:
Fixed